src/Implement/ResponseHandler.cpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021, 鍾淯丞, 周杰仕, 林仁鴻. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <iostream>
+#include <sys/socket.h>
+
+#include "OperateData.h"
+#include "ResponseHandler.h"
+#include "ResponseEncoder.h"
+
+void ResponseHandler::Send(const int &for_client_fd, std::unique_ptr<OperateData> &data) {
```

Comment:
consider use "const" on std::unique_ptr

Reply:
Please see the comment below on unique_ptr and ownership.